data/map.txt

```diff
@@ -29173,6 +29332,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The few settlements on the planet lie deep underground, far from any fault lines. Prone to hurricane-induced flooding, vast pumping systems in constant need of repair remove excess water from the cities. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
+	description `Most of the inhabitants of this planet are Syndicate dissidents who mainly support themselves through manufacturing. The Syndicate often destroys pirate factories at their borders, but their fleets do not find the monetary cost of venturing as far as New Tortuga worth the benefit. Here, shipyards, factories, and outfitters provide a continuous supply of goods and services for the system.`
+	spaceport `As you walk through the spaceport, scattered amidst the bars and black-markets, you notice a surprising number of religious buildings. There are a variety of churches, synagogues, mosques, temples, and others you can't quite remember the names of. Most of the shops are run not by pirates, but by ordinary folk, many wearing religious garb. The citizens here interact with pirates with friendliness and warmth, rather than the hostility and fear you see on other pirate worlds.`
+	spaceport `The relative peace is frequently broken by a pirate band wandering through the spaceport. While perhaps an auditory nuisance, the pirates here are generally more obedient and disciplined than those seen on other planets. Their ships are more advanced too, judging by the armaments and engines you noticed in the hangar.`
```

Comment:
I want to say that thematically the main spaceport (though no doubt not the only one) would likely be a thing unto itself, linked to the nearest cities by underground rail rather than actually attached to a single settlement. I imagine certain pirates and such have their own sections of the spaceport all to themselves in some regard.
However, these are already a pair of long descriptions so I don't think it's worth it trying to squeeze much else in.

Reply:
I don't see why that would be the case at all.  Maybe on a Republic world that may be the case, but I don't think Pirates care too much about zoning law.

Reply:
It's not about law it's about practicality - if things are largely underground, then the bigger any single thing is, the harder it is to make it safe - a large spaceport and a large city predominantly underground in the same place is a recipe for the whole lot of them collapsing, regardless of distance from any fault lines. 
Since the spaceport isn't actually explicitly linked to any particular city though, this is more a general observation.

Reply:
Hm.  Well, with your edit to the description, the spaceport doesn't have to be underground, it can be on the surface.  Which is probably a more practical place to land ships anyway.